rpa-builder/6.2/modules/ROOT/pages/toolbox-aws-operations-queries-with-aws.adoc

```diff
@@ -0,0 +1,111 @@
+= Queries With AWS
+
+The Queries with AWS Action Step enables you to extract specific data from a document by specifying a query parameter so that the response contains the data you need, without having to apply further filtering on the JSON Output to get the relevant values. 
```

Comment:
**suggestion**: The "specialty" of Queries with AWS is, that the query can/should be in natural language. This should be mentioned in some way. Otherwise the user might not know what "query parameters" to specify.